src/Components/Configuration/ConfigurationLoader.php

```diff
@@ -71,7 +71,13 @@ public function load(string $configFilename)
                 switch ($nodeType) {
                     case 'file':
 
-                        $fileName = (string)realpath(dirname($configFilename) . '/' . $nodeValue);
+                        $configuredFileName = dirname($configFilename) . '/' . $nodeValue;
```

Comment:
oh this is sooo good
i already planned moving those checks to the framework, away from the loader implementations

Reply:
Am I reading your mind? :)

Reply:
could be :D